CHANGES.rst

```diff
@@ -1,7 +1,18 @@
 Changes
 =======
 
-Version 0.5.0 (2018-04-23)
+Version 0.6.0 (2019-12-20)
+--------------------------
+
+- Fixes bug with big file uploads by using data streaming.
+- Adds user login endpoints using OAuth, currently configured to work with CERN
+  SSO but extensible to use other OAuth providers such as GitHub, more in `Invenio-OAuthClient <https://invenio-oauthclient.readthedocs.io/en/latest/>`_.
+- Adds endpoints to integrate with GitLab (for retrieving user projects and creating webhooks). GitLab instance to connect to configurable through environment variable ``REANA_GITLAB_HOST``.
```

Comment:
Second sentence might need some rephrasing. Could be something similar to `GitLab instance can be connected through configurable environment variable ``REANA_GITLAB_HOST``.` Or just get rid of it, it might be too detailed for CHANGES.

Reply:
I think you are right, reading it again I feel it is too detailed, I will remove it.